packages/optimize/package.json

```diff
@@ -0,0 +1,38 @@
+{
+  "name": "@adobe/react-native-aepoptimize",
+  "version": "1.0.0-alpha.1",
+  "description": "Adobe Experience Platform support for React Native apps.",
+  "homepage": "https://aep-sdks.gitbook.io/docs/",
+  "license": "Apache-2.0",
+  "main": "./ts/index.ts",
+  "scripts": {
+    "cleanup": "rm -rf node_modules",
+    "tsc": "tsc"
+  },
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/adobe/aepsdk-react-native",
+    "directory": "packages/optimize"
+  },
+  "keywords": [
+    "react-native",
+    "adobe",
+    "adobe experience platform",
+    "ajo",
+    "optimize",
+    "ios",
+    "android"
+  ],
+  "author": {
+    "name": "Adobe Experience Platform SDK Team"
+  },  
+  "peerDependencies": {
+    "@adobe/react-native-aepcore": "^1.0.0-beta",
+    "@adobe/react-native-aepedge": "^1.0.0-beta",
+    "@adobe/react-native-aepedgeidentity": "^1.0.0-beta",
```

Comment:
@adobe/react-native-aepcore, @adobe/react-native-aepedge and @adobe/react-native-aepedgeidentity are GA now, the Optimize package do not need to depend on those `beta` versions.